pystruct/models/multilabel_svm.py

```diff
@@ -42,12 +44,32 @@ def __init__(self, n_labels=None, n_features=None, edges=None,
 
     def _set_size_joint_feature(self):
         # try to set the size of joint_feature if possible
-        if self.n_features is not None and self.n_states is not None:
-            if self.edges is None:
-                self.edges = np.zeros(shape=(0, 2), dtype=np.int)
+        self._set_edge_structure()
```

Comment:
I think this is not necessary. ``initialize`` should have been called before ``_set_size_joint_feature`` If ``initialize`` has not been called, it's very likely that ``self.n_features`` and ``self.n_states`` is unknown yet.